cumulus/tasks/parse-pdr/tests/parse_pdrs_test.js

```diff
@@ -1,22 +1,13 @@
 'use strict';
 
+const aws = require('@cumulus/common/aws');
 const test = require('ava');
-const proxyquire = require('proxyquire');
```

Comment:
@yjpa7145 I think we can also remove this as a dependency in the package.json

Reply:
Fixed in 84fee36.